src/java.desktop/macosx/classes/apple/laf/JRSUIUtils.java

```diff
@@ -39,40 +39,55 @@
     static boolean isBigSurOrAbove = isMacOSXBigSurOrAbove();
```

Comment:
This is not required..Need to be removed.

Reply:
> This is not required..Need to be removed.

Done.